Assets/Scripts/PowerUps/TextChanger.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TextChanger : MonoBehaviour
+{
+    public TextMesh text; 
+    private const string SPEED_INDICATION = "Speed Boost!\n";
+    private const string JUMP_INDICATION = "Jump Boost\n";
+    private const string NO_INDICATION = "\n";
```

Comment:
Just a nitpick: reverse the name such that the common prefix is in the front (i.e., `INDICATION_SPEED`)

Reply:
Okie doe

Reply:
Fixed naming convention for now!